charts/posthog/values.yaml

```
@@ -1337,6 +1338,12 @@ pgbouncer:
   ## -- PgBouncer pod(s) annotation.
   podAnnotations: {}
 
+pgbouncer:
+  <<: *_pgbouncer
+
+pgbouncer_read:
```

Maybe for consistency:

```suggestion
pgbouncerRead:
```

Although I don't know what the generally used format is in the community.

---

Ahh good point, will change it